src/views/landing/ExperiencedPlayerHome/ExperiencedPlayerHome.vue

```diff
@@ -63,21 +62,23 @@
   export default class ExperiencedPlayerView extends Vue {
     @Prop({}) pageData!: Object;
 
+    get achievementRoadmap() {
+      return this.pageData?.achievement_roadmap || [];
+    }
+
     get masteringEterna() {
-      return get(this.pageData, 'achievement_roadmap', [])
-        .filter((p: { key: string }) => (p.key as string).includes('side_quest'))
-        .filter(p => p.level === p.current_level + 1)
-        .map(p => ({
+      return this.achievementRoadmap
```

Comment:
Changing line 63 to `@Prop({required: true}) pageData!: {achievement_roadmap: AchievementItem[]};` makes all of the types here redundent.

Reply:
thanks